integration_tests/mockApis/prisoners.ts

```diff
@@ -8,12 +8,12 @@ export default {
   stubPrisoner: (prisoner: Prisoner): SuperAgentRequest =>
     stubFor({
       request: {
-        method: 'GET',
-        url: prisonerOffenderSearchPaths.prisoner.show({ prisonNumber: prisoner.prisonerNumber }),
+        method: 'POST',
+        url: prisonerOffenderSearchPaths.prisoner.search({}),
       },
       response: {
         headers: { 'Content-Type': 'application/json;charset=UTF-8' },
-        jsonBody: prisoner,
+        jsonBody: prisoner ? [prisoner] : [],
```

Comment:
Do we have a need to stub a not found prisoner?

Reply:
Yeah, otherwise wiremock complains that the endpoint doesn't exist. I've played around with this for a bit but think it's good enough for now to get in as-is. I also think it's quite nice to have the explicit `stubPrisoner(undefined)` in the test as it makes it clear what's going on. We could possibly revisit this in future though if we need